aea/manager/project.py

```diff
@@ -93,8 +93,7 @@ def load(
         cls,
         working_dir: str,
         public_id: PublicId,
-        is_local: bool = False,
-        is_remote: bool = False,
```

Comment:
I can see you didn't like this and therefore changed it. But the problem is you're breaking backwards compatibility for a vanity reason. So I suggest reverting it and then opening an issue so it can be considered when doing a major bump of the library.